### PR TITLE
BOSA21Q1-96: migrate test suite to 0.24 version

### PR DESCRIPTION
**Pull request by aokon**
Updated:

* fixed broken specs for decidim-admin,  decidim-core, decidim-verifications
* speed up the system tests via cuprite engine
* updated factories
* migrated rubocop setup to latest decidim version

### CODE REVIEW

File: lib/extends/decidim-admin/app/forms/decidim/admin/area_form.rb [hunk 1]
```diff
@@ -10,9 +10,7 @@ module AreaFormExtend
     attribute :logo
     attribute :remove_logo
 
-    validates :logo,
-              file_size: { less_than_or_equal_to: ->(_record) { Decidim.maximum_attachment_size } },
-              file_content_type: { allow: ["image/png"] }
+    validates :logo, passthru: { to: Decidim::Area }
```
Comment (aokon) [inline lib/extends/decidim-admin/app/forms/decidim/admin/area_form.rb:13]: @sergei-krylov for latest decidim version there is a common pattern, so I have migrated this part for our extension

**Comment by aokon** [comment]
@sergei-krylov I made separate commits per module, but I think we can squash them during the merging it.

File: app/uploaders/decidim/area_logo_uploader.rb [hunk 2]
```diff
@@ -5,6 +5,14 @@ module Decidim
   class AreaLogoUploader < ImageUploader
     process quality: Decidim.image_uploader_quality
 
+    def extension_allowlist
+      %w(png)
```
Comment (malbert) [inline app/uploaders/decidim/area_logo_uploader.rb:9]: shouldn't we add jpg ?

**Review by malbert — APPROVED** [review]

File: app/uploaders/decidim/area_logo_uploader.rb (hunk 2 above)
Comment (aokon) [reply app/uploaders/decidim/area_logo_uploader.rb:9]: In the spec originally there was only png, so this is a main question here.

Comment (malbert) [reply app/uploaders/decidim/area_logo_uploader.rb:9]: ok let's keep it like this then

**Pull request merged**